Eatery Blue/UI/PillButton/PillButtonView.swift

```diff
@@ -53,7 +53,7 @@ class PillButtonView: UIView {
         }
 
         imageView.snp.makeConstraints { make in
-            make.width.height.equalTo(24)
+            make.width.height.equalTo(16)
```

Comment:
Is this OK to be a constant?

Reply:
Ideally it should resize wrt the screen but it should be ok for the MVP